README.md

```diff
@@ -1,66 +1,54 @@
-# editorial-tools-pinboard
+# PinBoard 📌
 
-📌 PinBoard - the omnipresent 'content association' and chat tool.
+`the omnipresent 'content association' and chat tool.`
 
```

Comment:
It would be nice to include a more detailed description of what Pinboard is, maybe we can find something from one of the OKR presentations etc

Reply:
excellent point - addressed in https://github.com/guardian/pinboard/pull/180/commits/00b330168a543b773193d92cc659f8272f62c40e (following team discussion on how we should define Pinboard)